microsoft-edge/devtools-guide-chromium/storage/indexeddb.md

```diff
@@ -28,110 +28,123 @@ To view and change [IndexedDB](https://developer.mozilla.org/docs/Web/API/Indexe
+*  To refresh the data, view an object store and then click the **Refresh** (![Refresh](./indexeddb-images/reload-icon.png)) button.
+
+*  To refresh all data in an IndexedDB database, view a database and then click **Refresh database**:
 
-![View a database](./indexeddb-images/storage-application-storage-indexeddb-notes_db-notes_os-refresh-database.png)
+   ![The database view, with the refresh button](./indexeddb-images/refresh-db.png)
 
 
 <!-- ====================================================================== -->
 ## Edit IndexedDB data
 
-IndexedDB keys and values aren't editable from the **Application** tool.  However, since DevTools has access to page context, you can run JavaScript code within DevTools to edit IndexedDB data.
+IndexedDB keys and values aren't editable from the **Application** tool.  However, since DevTools has access to the page context, you can run JavaScript code within DevTools to edit the data stored in an IndexedDB database.
+
+#### Edit IndexedDB data with the Console tool
+
+1. In DevTools, on the Activity Bar, select the **Console** tab.
+
+1. In the **Console** tool, run JavaScript code to edit the IndexedDB data. For example, to add a new value to the `keyval` object store, run the following code:
+   
+   ```javascript
+   let connection = indexedDB.open("keyval-store", 1);
+
+   connection.onsuccess = e => {
+     const database = e.target.result;
+     const transaction = database.transaction("keyval", "readwrite");
+     const objectStore = transaction.objectStore("keyval");
+     const request = objectStore.add({foo: "bar"}, "new-key");
+     request.onsuccess = e => {
+       console.log(e.target.result);
+     }
+   }
+   ```
 
-### Edit IndexedDB data with Snippets
+#### Edit IndexedDB data with snippets
 
-[Snippets](../javascript/snippets.md) are a way to store and run blocks of JavaScript code within DevTools.  When you run a Snippet, the result is logged to the **Console**.  You can use a Snippet to run JavaScript code to edit an IndexedDB database.
+[Snippets](../javascript/snippets.md) are a way to store and run JavaScript code repeatedly, within DevTools.  If you need to edit IndexedDB data often, store it in a new snippet. To learn more, see [Run snippets of JavaScript on any webpage](../javascript/snippets.md).
 
-![Using a Snippet to interact with IndexedDB](./indexeddb-images/storage-sources-snippets-indexeddb-output.png)
+![Using a Snippet to interact with IndexedDB](./indexeddb-images/edit-from-snippet.png)
 
 
 <!-- ====================================================================== -->
 ## Delete IndexedDB data
 
-### Delete an IndexedDB key-value pair
+#### Delete an IndexedDB key-value pair
 
 1. [View an IndexedDB object store](#view-indexeddb-data).
 
-1. Click the key-value pair that you want to delete.  DevTools highlights it to indicate that it is selected.
+1. Click the key-value pair that you want to delete.  DevTools highlights it to indicate that it is selected:
```

Comment:
```suggestion
1. Click the key-value pair that you want to delete.  DevTools highlights the key-value pair to indicate that it is selected:
```
too many "it" in sentence; sentence doesn't contain a word to resolve "it", so sentence is meaningless on its own.